src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,29 @@
+package duke;
+
+/**
+ *  DukeException class encapsulates information about a custom exception for Duke
+ *  and inherits functionality from the Exception class
+ */
+public class DukeException extends Exception {
+    private static final String TOP_BORDER = "_____________________________________________________________________";
+    private static final String BTM_BORDER = "---------------------------------------------------------------------";
+    private static final String PADDING = "  ";
+
+    /**
+     * Create a new DukeException to be thrown
+     *
+     * @param msg The error message
+     */
```

Comment:
Hi, I believe there should be punctuation at the end of the description, based on the javadocs